setup_ctp7.sh

```diff
@@ -123,29 +133,55 @@ then
         ln -sf recover_v2.sh scripts/recover.sh
     fi
     # echo "creating links for CTP7 firmware version: ${ctpfw}"
-    if [ ! -f "fw/gem_ctp7_gem_ctp7_v${ctp7fw//./_}.bit" ]
-    then
-        echo "CTP7 firmware fw/gem_ctp7_v${ctp7fw//./_}.bit missing, downloading"
-        echo "wget https://github.com/evka85/GEM_AMC/releases/download/v${ctp7fw}/gem_ctp7_v${ctp7fw//./_}_${nlinks}oh.bit -O fw/gem_ctp7_v${ctp7fw//./_}_${nlinks}oh.bit"
-        wget https://github.com/evka85/GEM_AMC/releases/download/v${ctp7fw}/gem_ctp7_v${ctp7fw//./_}_${nlinks}oh.bit -O fw/gem_ctp7_v${ctp7fw//./_}_${nlinks}oh.bit
-    fi
-
-    echo "ln -sf fw/gem_ctp7_v${ctp7fw//./_}_${nlinks}oh.bit fw/gem_ctp7.bit"
-    ln -sf gem_ctp7_v${ctp7fw//./_}_${nlinks}oh.bit fw/gem_ctp7.bit
+    if [ -n "${usege21}" ]
```

Comment:
Can these two block be merged with some `ge21suffix` variable that gets inserted into the various files as needed, e.g.,
```bash
ge21suffix='_ge21'
wget https://github.com/evka85/GEM_AMC/releases/download/v${ctp7fw}/gem_ctp7_v${ctp7fw//./_}${ge21suffix}_${nlinks}oh.bit -O fw/gem_ctp7_v${ctp7fw//./_}${ge21suffix}_${nlinks}oh.bit"
```

Reply:
indeed, please see the revised code